cedar-lean/Cedar/Thm/Data/List.lean

```diff
@@ -1158,40 +1486,7 @@ theorem mem_of_sortedBy_unique {α β} [LT β] [StrictLT β] [DecidableLT β] [D
       specialize hlt x hx
       simp only [hf, StrictLT.irreflexive] at hlt
 
-/-! ### map -/
-
-/-
-These are copied from Mathlib. We can delete them if they get added to Std.
--/
-
-theorem map_pmap_subtype
-  {p : α → Prop}
-  (f : α → β)
-  (as : List α)
-  (h : ∀ a, a ∈ as → p a)
-  : List.map (fun x : { a : α // p a } => f x.val) (List.pmap Subtype.mk as h)
-    =
-    List.map f as
```

Comment:
Moved to the `map` section (instead of a dedicated section for "copied from Mathlib")